include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -8281,12 +8348,19 @@ FEEvaluation<dim,
              Number,
              VectorizedArrayType>::
   evaluate(const VectorizedArrayType *            values_array,
-           const EvaluationFlags::EvaluationFlags evaluation_flags)
+           const EvaluationFlags::EvaluationFlags evaluation_flag)
```

Comment:
Don't we normally write flags (plural)?

Reply:
This was the only place where plural was used. I changed it to singular to be consistent.